mindsdb/integrations/handlers/paypal_handler/paypal_handler.py

```diff
@@ -38,6 +38,9 @@ def __init__(self, name: str, **kwargs):
 
         invoices_data = InvoicesTable(self)
         self._register_table("invoices", invoices_data);
+    
+        subscripotion_data = SubscriptionTable(self)
```

Comment:
There is a typo in this variable name - it should be **subscriptions_data** throughout the code.